README.md

```diff
@@ -18,7 +18,7 @@
 
 # Vets Who Code Web App
 
-The Vets Who Code Web App is a Jamstack Applicaton designed for speed and to ease the learning curve of our veterans contributing to open source.
+The Vets Who Code Web App is a Next.JS Application designed for developer productivity as well as a well thought out app to ease the learning curve of of veterans learning to contribute to open source.
```

Comment:
Please update "learning curve of of veterans" to "learning curve of veterans".

Secondly, what updates did you run for the package-lock.json? Just interested to know.

Reply:
I just followed the install directions. I thought I changed that. I'll fix it later.

Reply:
@Cameron-Porter I did NVM install and NPM install